dll/cpl/input/lang/en-US.rc

```diff
@@ -1,19 +1,23 @@
 LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
 
 IDD_PROPPAGESETTINGS DIALOGEX 0, 0, 254, 228
-STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Settings"
+STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
```

Comment:
In all those translations, please keep the "STYLE" line before the "CAPTION".

Reply:
Fixed.